pythermalcomfort/models/two_nodes_gagge_ji.py

```
@@ -0,0 +1,455 @@
+        for key in first_result_dict.keys():
+            output_data[key] = []
+
+        for res_dict in results_array_of_dicts:
+            for key, value in res_dict.items():
+                # creates a list of NumPy arrays for each key
+                output_data[key].append(value)
+
+    return GaggeTwoNodesJi(**output_data)
+
+
+def _two_nodes_ji_optimized(
```

Pylint (3.3.7)

[refactor] Too many arguments (15/5)

(R0913)